resizable-circle/demo.js

```diff
@@ -0,0 +1,134 @@
+/**
+ * Adds resizable geo shapes to map
+ *
+ * @param {H.Map} map                      A HERE Map instance within the application
+ */
+function createResizableCircle(map) {
+  var circle = new H.map.Circle(
+        {lat: 50, lng: 22.8},
+        85000,
+        {
+          style: {fillColor: 'rgba(250, 250, 0, 0.7)', lineWidth: 0}
+        }
+      ),
+      circleOutline = new H.map.Polyline(
+        circle.getGeometry().getExterior(),
+        {
+          style: {lineWidth: 8, strokeColor: 'rgba(255, 0, 0, 0)'}
+        }
+      ),
+      circleGroup = new H.map.Group({
+        volatility: true, // mark the group as volatile for smooth dragging of all it's objects
+        objects: [circle, circleOutline]
+      }),
+      circleTimeout;
+
+  // ensure that the objects can receive drag events
+  circle.draggable = true;
+  circleOutline.draggable = true;
+
+  // extract first point of the circle outline polyline's LineString and
+  // push it to the end, so the outline has a closed geometry
+  circleOutline.getGeometry().pushPoint(circleOutline.getGeometry().extractPoint(0));
+
+  // add group with circle and it's outline (polyline)
+  map.addObject(circleGroup);
+
+  // event listener for circle group to show outline (polyline) if moved in with mouse (or touched on touch devices)
+  circleGroup.addEventListener('pointerenter', function(evt) {
+    var group = this,
+        circleOutline = group.getObjects()[1],
```

Comment:
Why can't the cricleOutline from the outer scope be used?

Reply:
done